src/ring_theory/tensor_product.lean

```diff
@@ -714,6 +714,46 @@ end
 
 end monoidal
 
-end tensor_product
+section
+
+variables {R A B S: Type*} [comm_semiring R] [semiring A] [semiring B] [comm_semiring S]
+variables [algebra R A] [algebra R B] [algebra R S]
+variables (f : A →ₐ[R] S) (g : B →ₐ[R] S)
+
+variables (S)
```

Comment:
I'd argue `R` should be implicit not `S`, to match `algebra.lmul'`

Reply:
Hmmm. I understand that matching `algebra.lmul'` would be better.
But why was `algebra.lmul'` implicit in `S` at the first place?
The map seems to depend on `S` quite heavily, and it should be able to infer `R` from `S` easier.

Reply:
You can't infer `R` from `S`. You can infer `S` from `a ⊗ₜ b` but not `R`, which is why `R` is explicit in the other def.